### ISSUE TITLE
export command creates different output for the same input

### ISSUE THREAD

**Issue by elifarley** [rethinkdb/rethinkdb#3911]
The command below was executed on 2 different hosts:

```
rethinkdb export -e test.txstats --format csv --fields method,count,sum,min,avg,max,s2 -d /tmp/txstats-export
```

The result for the first host is:

```
method,count,sum,min,avg,max,s2
MyClass.myMethod,154,56334,161,365.8051948051948,1260,258.7894411965032
```

But for the second host we have:

```
b'method',b'count',b'sum',b'min',b'avg',b'max',b's2'
b'MyClass.myMethod',b'154',b'56334',b'161',b'365.8051948051948',b'1260',b'258.7894411965032'
```

**Comment by Tryneus** [comment]
I suspect this is a problem between the script's `python2`/`python3` compatibility.  Looking into this today.

**Comment by elifarley** [comment]
Meanwhile, this workaround will help:

```
sed -e "s/',b'/,/g; s/^b'//; s/'//"
```

**Comment by Tryneus** [comment]
The fix is up in review 2701, thanks for reporting this, @elifarley!

**Comment by Tryneus** [comment]
The fix has been approved and merged into `next` in commit 2c8de27, cherry-picked into `v2.0.x` in commit ada9308, and cherry-picked into `v1.16.x` in commit 3ede609.

Tested with Python 2.6, 2.7, 3.1, 3.2, 3.3, and 3.4, and verified that all CSV outputs were identical.

@atnnn, could we do a python driver release for this?

**Comment by AtnNn** [comment]
A new version of the Python driver has been released with the fix, as version 1.16.0-3
